Automattic/wp-calypso
# Partner Program: add empty signup page in dev environment

Related to Automattic/jetpack-manage#49.

## Proposed Changes

This adds a mostly-empty signup page at `/partner-program/signup`. It largely copies the current agency sign-up code (added in #63609).

Down the road it'd be good to abstract the two, but at the moment the other form is being modified as well, so best to do that after the dust has settled a bit.

## Testing Instructions



Using the `add/jp-manage/49-new_partner_program_signup_page` branch:
1. Verify this page loads: http://jetpack.cloud.localhost:3000/partner-program/signup
2. In a private browser window, go here: http://jetpack.cloud.localhost:3000/partner-program/signup
3. Confirm you are prompted to log in.
4. Log in, and confirm you are redirected to the same page as in step 1, which looks like this:

![image](https://github.com/Automattic/wp-calypso/assets/32492176/29fdf4fd-bb02-4000-b942-e76486503dae)

Note that Automattic/jetpack-manage#50 and Automattic/jetpack-manage#51 will handle the form contents/functionality, after which we'll need to enable the section in the various configs in preparation for production. Tests should be added down the road.
